presto-docs/src/main/sphinx/language/types.rst

```diff
@@ -65,12 +65,22 @@ Floating-Point
 A real is a 32-bit inexact, variable-precision implementing the
 IEEE Standard 754 for Binary Floating-Point Arithmetic.
 
+It strays from the IEEE standard when it comes to handling of nans.
```

Comment:
```suggestion
Presto strays from the IEEE standard when handling NaNs.
```
Is "Presto" correct? I found "It" vague. Also, consider this minor rephrasing suggestion for conciseness.